tests/integration/targets/nxos_static_routes/tests/common/____overridden.yml

```diff
@@ -43,7 +43,6 @@
           - "'no ip route 192.0.2.64/28 192.0.2.23 name merged_route 1' in result.commands"
```

Comment:
Why are we renaming this file?

Reply:
If overridden doesn't remove things that are specified in the task, why would this be a problem? I'm trying to understand the implications of running a valid overridden task by end user. If default route is part of the task, shouldn't it be left untouched in any circumstance?

Reply:
Yes, to answer your question the reason is similar to why we removed delete all
For a configuration -
```
switch# show running-config | include '^ip(v6)* route'
switch# show running-config | section '^vrf context'
vrf context management
  ip name-server 192.168.255.1
  ip route 0.0.0.0/0 192.168.255.1
switch# 
```
And a play as such would be Idempotent!
```
    - name: The overridden story
      cisco.nxos.nxos_static_routes:
        config:
          - address_families:
              - afi: ipv4
                routes:
                  - dest: 0.0.0.0/0
                    next_hops:
                      - forward_router_address: 192.168.255.1
            vrf: management
        state: overridden
```
We cannot create a good playbook here as the `address_families.afi.routes.next_hops.forward_router_address`  might not be fixed in CI.